app.py

```diff
@@ -34,7 +36,11 @@ def process_request(self, req, resp):
 
 class PredictResource:
     def __init__(self):
-        self.model_wrapper = self._load_model()
+        try:
+            self.model_wrapper = self._load_model()
+        except ModuleNotFoundError as e:
+            logger.warning("No model module found. Using a dummy one.")
+            self.model_wrapper = RandomModel()
```

Comment:
I am not sure a random model as a fallback would be a good idea. This can cause a lot of problems if it remains undetected. Instead, I would stop the execution altogether with a descriptive and clear error message.

Reply:
Well, it's a fair point. I used a fallback model to be able to run the base image and make sure it works before debugging an image that is built on top of it. Otherwise, I had to build telesto-base image, then telesto-models image and run the telesto-models image to debug issues with telesto-base.
So far, we're manually evaluating and deploying models so it shouldn't be that easy to overlook the event of falling back to the random model.

Reply:
My main issue with this is if a user doesn't check the accuracy of the model, only that it works, this can go undetected. Anyway, I'll just open a new issue about this and will sort out later.

Reply:
I think I found a compromise in this commit https://github.com/telesto-ai/telesto-base/pull/15/commits/bcfad53b02b83b15ee7b27b89c88411d2f128a18

Reply:
Ok, I like the idea. In general, a DEBUG mode would be useful.